src/web3data.js

```diff
@@ -94,6 +94,18 @@ class Web3Data {
     this.websocket.off({eventName, filters}, callback)
   }
 
+  getGasPrice() {
+    return this.block.getGasPrice()
+  }
+
+  getBlockNumber() {
```

Comment:
@taylorjdawson weren't we going to add these under `.eth.*` instead? I think at this top level it doesn't quite make sense for all.
Same for the below 3 methods